assets/components/minishop2/js/mgr/product/update.js

```diff
@@ -119,13 +119,16 @@ Ext.extend(miniShop2.panel.UpdateProduct, miniShop2.panel.Product, {
             }
             var item = originals[i];
             if (item.id == 'modx-resource-tabs') {
+                // Get the "Resource Groups" tab to move it to the end
+                var resource_groups_tab = item.items.pop();
                 // Additional tabs
                 if (miniShop2.config['show_comments'] != 0) {
                     item.items.push(this.getComments(config));
                 }
                 if (miniShop2.config['show_gallery'] != 0) {
                     item.items.push(this.getGallery(config));

```

Comment:
```suggestion
                    item.items.splice(2, 0, this.getGallery(config));
```

Лучше сделай вот так, а свой добавленный код убери